models/meshmodel/registry/registry.go

```diff
@@ -440,6 +465,22 @@ func (rm *RegistryManager) GetRegistrant(e Entity) Host {
 	_ = rm.db.Where("id = ?", reg.RegistrantID).Find(&h).Error
 	return h
 }
+func (rm *RegistryManager) GetRegisteriesCount() (v1alpha1.EntitySummary, error) {
+
+	var counts v1alpha1.EntitySummary
+	db := rm.db
+
+	query := db.Table("registries").
```

Comment:
Please confirm that these counts are both per Registrant and aggregate total for the whole Registry.

Reply:
Removed the old function because we already had a function that return same values